tests/Model/PHPClass.php

```diff
@@ -59,7 +56,7 @@ public function readObjectFromStubNode($node): self
     {
         $this->name = $this->getFQN($node);
         $this->collectLinks($node);
-        $this->collectSinceDeprecatedVersions($node);
+        $this->collectSinceRemovedDeprecatedVersions($node);
```

Comment:
Let's check `@since` as well for the consistency

Reply:
Such test already exists